src/battle/battle.cpp

```diff
@@ -80,16 +80,30 @@ void Battle(MainCharacter& main_character, std::vector<Creature>& foes)
 			if (combatants[i].first == main_character_ptr)
 			{
 				std::cout << "Main character attacks!\n";
+				std::cout << "select a target\n";
+				for (size_t i = 0; i < foes_ptrs.size(); i++)
+				{
+					std::cout << i + 1 << " " << foes_ptrs[i] << std::endl;
```

Comment:
`foes_ptrs[i]` is a pointer to a `Creature`. You have to call its `get_display_name()` method to actually get what you want.